lib/rules/id-blacklist.js

```diff
@@ -101,21 +104,21 @@ module.exports = {
                         // Report AssignmentExpressions only if they are the left side of the assignment
                     } else if (effectiveParent.type === "AssignmentExpression" &&
                         (effectiveParent.right.type !== "MemberExpression" ||
-                        effectiveParent.left.type === "MemberExpression" &&
-                        effectiveParent.left.property.name === node.name)) {
+                            effectiveParent.left.type === "MemberExpression" &&
+                            effectiveParent.left.property.name === node.name)) {
                         if (isInvalid(name)) {
                             report(node);
                         }
                     }
 
-                // Properties have their own rules
+                    // Properties have their own rules
                 } else if (node.parent.type === "Property") {
 
                     if (shouldReport(effectiveParent, name)) {
                         report(node);
                     }
 
-                // Report anything that is a match and not a CallExpression
+                    // Report anything that is a match and not a CallExpression
```

Comment:
Mind undoing this? This is referring to the follow block and I think it's clearer to have it indented as it was.

Reply:
Yeah, I see that. Strange :/ Seems like something on GitHub's end?

Reply:
Seems like it. This is the first time I've had this happen. 🤷‍♂